src/main/java/com/mkopec/ticketbookingapp/init/SqlFunction.java

```diff
@@ -0,0 +1,23 @@
+package com.mkopec.ticketbookingapp.init;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+
+public class SqlFunction {
+    public static void ticketInsert(Connection connection, String row, String numbers,
```

Comment:
nieużywane...

Reply:
IDE nie podświetla tego, ale jest używane w skrypcie data.sql. Stworzyłem taką funkcję do wprowadzania już już zarezerwowanych biletów (w skrypcie byłby to dosyć rozbudowany sql, a w skryptach sql dla H2 nie można zdefiniować funkcji, więc napisałem metodę w javie i wywołuje ją z poziomu skryptu)

Reply:
ok, to ja jestem głupi jednak w te SQLe strasznie ;)